src/main/java/BulkInsertApp.java

```diff
@@ -0,0 +1,69 @@
+import java.io.InputStream;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.PreparedStatement;
+import java.sql.SQLException;
+import java.util.Map;
+
+import org.yaml.snakeyaml.Yaml;
+
+import com.beust.jcommander.JCommander;
+import com.beust.jcommander.Parameter;
+
+public class BulkInsertApp {
+
+    public static final int USER_ID = 1;
+    public static final String APPLICATION_YML = "/application.yml";
+
+    @Parameter(names = {"--count"}, description = "Number of records to insert", required = true)
+    private int count;
+
+    public static void main(String[] args) {
+        BulkInsertApp app = new BulkInsertApp();
+        JCommander.newBuilder().addObject(app).build().parse(args);
+        app.run();
+    }
+
+    private void run() {
+        String insertSQL = "INSERT INTO boards"  + " (title, content, user_id) VALUES (?, ?, ?)";
+
+        Map<String, Object> config = yamlConfigReader();
+        Map<String, String> dataSource = (Map<String, String>) config.get("datasource");
+        final String url = dataSource.get("url");
+        final String user = dataSource.get("user");
+        final String password = String.valueOf(dataSource.get("password"));
+
+        try (Connection connection = DriverManager.getConnection(url, user, password)) {
+            connection.setAutoCommit(false);
+            try (PreparedStatement preparedStatement = connection.prepareStatement(insertSQL)) {
+                for (int i = 1; i <= count; i++) {
+                    preparedStatement.setString(1, "test_title" + i);
+                    preparedStatement.setString(2, "test_content" + i);
+                    preparedStatement.setInt(3, USER_ID);
+                    preparedStatement.addBatch();
```

Comment:
Why did you not use INSERT bulk query? I think bulk insert is more efficient.
Please check what is the difference between bulk INSERT query and N INSERT queries with batch.
And fix this to use a single bulk INSERT query per 1000 items.

Reply:
Bulk query faster than using separate single-row [INSERT] statements. I found it in the following mysql document. 
[https://dev.mysql.com/doc/refman/8.0/en/insert-optimization.html](url)

According to the document, sending one large statement will save almost 7 per insert statement.